app-misc/ledit/ledit-2.04.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit eutils
+
+DESCRIPTION="A line editor to be used with interactive commands"
+SRC_URI="http://pauillac.inria.fr/~ddr/ledit/distrib/src/${P}.tgz"
+HOMEPAGE="http://pauillac.inria.fr/~ddr/ledit/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+ocamlopt"
+RESTRICT="installsources !ocamlopt? ( strip )"
+
+DEPEND=">=dev-lang/ocaml-3.09:=[ocamlopt?]
+	dev-ml/camlp5:="
+RDEPEND="${DEPEND}"
+
+src_compile() {
+	emake -j1 all
```

Comment:
Could you verify this is still needed?

Reply:
I guess you've meant `-j1`, right?

Reply:
Verified by invoking `ebuild ledit-2.04.ebuild install`. Is it enough?

Reply:
Yep, I meant the `-j1` part. They are usually remnants of the past.

If it compiles and installs fine on higher MAKEOPTS then it's confirmed to work.